frameworks/elastic/tests/test_custom_options.py

```diff
@@ -0,0 +1,29 @@
+from tests import config
+import pytest
+import sdk_cmd
+import sdk_install
+import sdk_tasks
+import logging
+log = logging.getLogger(__name__)
+
+
+@pytest.mark.sanity
+def test_xmx_and_xms_flags():
+    sdk_install.uninstall(config.PACKAGE_NAME, config.SERVICE_NAME)
+    #installing elastic service and passing customized json to overwrite default values.
+    sdk_install.install(
+        config.PACKAGE_NAME,
+        config.SERVICE_NAME,
+        config.DEFAULT_TASK_COUNT
```

Comment:
As we discussed in the [comments](https://github.com/mesosphere/dcos-commons/pull/2975#discussion_r265553056), this test should install Elastic with non-default values for:

- `master_nodes.heap`
- `data_nodes.heap`
- `coordinator_nodes.heap`
- `ingest_nodes.heap`

E.g.:
```
master_nodes_heap = 2000
data_nodes_heap = 3000
coordinator_nodes_heap = 4000
ingest_nodes_heap = 5000

sdk_install.install(
    config.PACKAGE_NAME,
    config.SERVICE_NAME,
    config.DEFAULT_TASK_COUNT,
    {
      "master_nodes: {
        "heap": master_nodes_heap
      },
      "data_nodes: {
        "heap": data_nodes_heap
      },
      "coordinator_nodes: {
        "heap": coordinator_nodes_heap
      },
      "ingest_nodes: {
        "heap": ingest_nodes_heap
      },
    },
)
```

And then verify that the outputs for each particular pod task [has the correct value](https://github.com/mesosphere/dcos-commons/pull/2975#discussion_r265553056) inside the loop.

Reply:
Done!